docs/content/ingestion/index.md

```diff
@@ -244,6 +246,9 @@ for the `comment` column.
 }
 ```
 
+## metricsSpec
+ The `metricsSpec` is a list of [aggregators](../querying/aggregations.html). If `rollup` is false in the granularity spec, the metrics spec can be an empty list and all columns can be defined in the dimensionsSpec instead.
```

Comment:
Is it 'can be an empty list' or 'should be (must be?) an empty list'? Same with 'columns can be defined'

Reply:
How about backticking `dimensionsSpec`?

Reply:
It's optional, but "more proper" to put it in the `dimensionSpec` (since with no rollup there's not a real concept of 'metric'). 

Going forward, I think we should re-do how the ingestion specs are represented so the rollup case -> define dimensions/metrics, no rollup -> define columns

Reply:
Updated to backtick `dimensionsSpec`, changed the comment to say "should" with the reasoning, but mention that it's optional.